scala/private/macros/scala_repositories.bzl

```diff
@@ -4,7 +4,30 @@ load(
     _default_maven_server_urls = "default_maven_server_urls",
 )
 load("//third_party/repositories:repositories.bzl", "repositories")
-load("@io_bazel_rules_scala_config//:config.bzl", "SCALA_MAJOR_VERSION")
+load("@io_bazel_rules_scala_config//:config.bzl", "SCALA_MAJOR_VERSION", "SCALA_VERSION")
+
+def dt_patched_compiler_setup():
+    patch = "@io_bazel_rules_scala//dt_patches:dt_compiler_%s.patch" % SCALA_MAJOR_VERSION
+
+    if SCALA_MAJOR_VERSION == "2.12":
+        minor_version = int(SCALA_VERSION[SCALA_VERSION.find(".", 2) + 1:])
```

Comment:
Maybe `SCALA_VERSION.split(".")[2]` would be cleaner? And maybe this can be moved to `@io_bazel_rules_scala_config//:config.bzl` to keep version parsing in single place.

Reply:
done